specification/sql/resource-manager/Microsoft.Sql/2017-03-01-preview/managedDatabases.json

```diff
@@ -0,0 +1,556 @@
+{
+  "swagger": "2.0",
+  "info": {
```

Comment:
Please ensure that schema repo is updated with this and these new APIs are exposed through the /operations API. @jaredmoo will have context on this if needed.